src/bpmncwpverify/builder/cwp_builder.py

```diff
@@ -62,7 +62,7 @@ def _add_and_check_expressions(
                 if not edge or not (parent_id_ref := mx_cell.get("id")):
                     raise Exception("Parent edge not found or no parent ID reference")
 
-                edge.expression = edge.cleanup_expression(expression)
+                edge.expression = expression
                 result = expr_checker.build(edge.expression, self.symbol_table)
```

Comment:
Does `build` make sense for this interface? It doesn't build anything, rather it type checks the expression. I might suggest that `type_check` makes more sense.

Let's rename it in a PR that follows this one.